### DISCUSSION TITLE
How to add tolerations in csi-cephfsplugin daemonset?

### DISCUSSION THREAD

**Discussion by Dev-Destructor** [rook/rook#14423] [Q&A]
I am trying to add tolerations in csi-cephfsplugin daemonset using CephCluster CR. However, the tolerations I provide in the CR are not getting added to the daemonset. Can I have an example configuration?

**Comment by travisn** [comment]
The CSI placement settings such as tolerations are currently specified in the operator.yaml, rather than the CephCluster CR. See the example [here](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L258-L263) for setting the cephfs plugin tolerations.

**Comment by Dev-Destructor** [comment]
Okay thanks for the info...
